larq_zoo/sota/quicknet.py

```diff
@@ -104,6 +169,21 @@ def build(self) -> tf.keras.models.Model:
 
         model = tf.keras.Model(inputs=self.image_input, outputs=x, name="quicknet")
 
+        return model
+
+
+@factory
+class QuickNetFactory(QuickNetBaseFactory):
+    """Quicknet - A model designed for fast inference using [Larq Compute Engine](https://github.com/larq/compute-engine)"""
+
+    spec = Field(
+        lambda: ([2, 3, 4, 4], [64, 128, 256, 512], [False, False, False, False])
```

Comment:
I am not sure if we should really have this as a field, or if we should just make the total number of layers configurable. I don't have a strong opinion about this, but for me it is not really intuitive what the different config values are here.

Reply:
Yes I'm also not too happy about this. We could split it into `blocks_per_section`, `section_filters`, `use_attention_in_section` or something like that?